Automattic/wp-calypso
# Add tests for new credit card management flow

#### Changes proposed in this Pull Request

* This PR adds a few test cases to components, selectors, reducers, and actions to the code which was added as a part of new credit card management flow

`Note` Please run `yarn` or `yarn install` if you see the tests are failing due to missing modules

#### Testing instructions

* Run `git checkout update/add-tests-for-new-credit-card-management-flow && git pull`
* Since we have added tests under `client/jetpack-cloud/sections/partner-portal` run `yarn run test-client client/jetpack-cloud/sections/partner-portal` to run the tests. 
* Make sure all the tests are passing

Related to 1201734780767770-as-1201829766632184
